cloud/wfm/src/JsonSchema.ts

```diff
@@ -0,0 +1,155 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2016 Richard Adams (https://github.com/enriched)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+// retrieved from https://gist.github.com/enriched/c84a2a99f886654149908091a3183e15
+
+interface JsonSchema {
```

Comment:
Did you meant runtime? 
IMHO validation is separate concern that we can provide this as extension to Step definition framework. I'm still not convinced why we need that file :)

The way I see it is that actual step developers can decide to validate their data before sending it as result. That's the best place. We can provide frameworks for that or even generate it like here: https://www.npmjs.com/package/typescript-json-schema